ios/RCTWebRTC/VideoCaptureController.h

```diff
@@ -6,6 +6,7 @@
 #import "CaptureController.h"
 
 @interface VideoCaptureController : CaptureController
+@property (nonatomic, readonly, strong) RTCCameraVideoCapturer *capturer;
```

Comment:
This should likely be the base class, so effects can be applied to any capturer.

Reply:
Left for now but can be changed if needed.